nusight2/src/client/components/vision/camera/visual_mesh.ts

```diff
@@ -1,12 +1,8 @@
 import * as THREE from 'three'
 import { Vector2 } from '../../../math/vector2'
-import { bufferGeometry } from '../../three/builders'
-import { rawShader } from '../../three/builders'
-import { shaderMaterial } from '../../three/builders'
-import { mesh } from '../../three/builders'
+import { bufferGeometry, mesh, rawShader, shaderMaterial } from '../../three/builders'
```

Comment:
I am not qualified to review typescript lol.

Reply:
Thats ok, Im not qualified to write typescript, but this happened from some formatter or other